pyproject.toml

```diff
@@ -20,18 +20,18 @@ black = "^22.8.0"
 pre-commit = "^2.20.0"
 
 
-[tool.poetry.group.extras.dependencies]
-gpytorch = "^1.9.0"
-salib = "^1.4.6.1"
+# [tool.poetry.group.extras.dependencies]
+# gpytorch = "^1.9.0"
+# salib = "^1.4.6.1"
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.poetry.extras]
```

Comment:
We should leave the gpflow and gpytorch extra dependencies in until we are ready to have separate packages with support for gpflow and gpytorch.